supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -487,6 +487,17 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 
 *See also*:
 
+[discrete]
+[[codeready-linux-builder-repository]]
+==== image:images/caution.png[with caution] CodeReady Linux Builder (CRB) repository (noun)
```

Comment:
If the term should never be expanded, should the abbreviation be used in the title? Writers can still find this glossary term if they CTRL+F "CRB" because of the Incorrect forms entry.

Reply:
Okay, removing "CRB"